converters/fromZigbee.js

```diff
@@ -2420,6 +2420,12 @@ const converters = {
         convert: (model, msg, publish, options, meta) => {
             const dpValue = tuya.firstDpValue(msg, meta, 'moes_switch');
             const dp = dpValue.dp;
+
+            // tuya_switch datapoints
+            if (dp >= 1 && dp <= 4) {
```

Comment:
Why was this added, this prevents `fromZigbee:moes_switch: NOT RECOGNIZED DP` from being logged.

Reply:
If not to add this condition when the state changes it also displays warning `fromZigbee:moes_switch: NOT RECOGNIZED DP`. So `tuya_switch` and `moes_switch` converters work together at the same time.
I'm sure that it's solution not good, but it works fine. Maybe it makes sense to replace switch case to if conditions.

Reply:
thanks for the explanation!